plugins/virtualenvwrapper/virtualenvwrapper.plugin.zsh

```diff
@@ -1,55 +1,28 @@
-virtualenvwrapper='virtualenvwrapper.sh'
-virtualenvwrapper_lazy='virtualenvwrapper_lazy.sh'
+function {
+    # search in these locations for the init script:
+    for f in $commands[virtualenvwrapper_lazy.sh] \
+             $commands[virtualenvwrapper.sh] \
+             /usr/share/virtualenvwrapper/virtualenvwrapper{_lazy,}.sh \
+             /usr/local/bin/virtualenvwrapper{_lazy,}.sh \
+             /etc/bash_completion.d/virtualenvwrapper \
+             /usr/share/bash-completion/completions/virtualenvwrapper
+    do
+        [[ -f $f ]] || continue
+        source $f
+        return
```

Comment:
You can use `break` instead so that you can remove the self-executing anonymous function.

Reply:
Hi, can you explain this comment? If a break were used here it would result in the error message being displayed, incorrectly.

Reply:
I didn't see that, you're right.